docs/source/events/examples/payment-status-change.yaml

```diff
@@ -0,0 +1,21 @@
+id: 123
+type: payment_status_change
+version: 1.1.0
+# Payment ID. Use this to match multiple status changes for the same payment.
+pid: '9088'
+# Machine name of the payment method
+controller: stripe_payment_credit_card
+# Labels for the payment method
+method_generic: Credit card
+method_specific: Stripe Payment (Live)
+# New and old payment status
+status: stripe_payment_status_accepted
```

Comment:
Maybe link to somewhere we we document some of the payment statuses?
https://support.impact-stack.org/hc/en-us/articles/11705625993244

Reply:
The main article about this event type (where this example is embedded) has explanations for some of the statuses.